test/reporters/tap.js

```diff
@@ -179,3 +180,18 @@ test('skip test', function (t) {
 	t.is(actualOutput, expectedOutput);
 	t.end();
 });
+
+test('reporter strips ANSI characters', function (t) {
+	var reporter = tapReporter();
+
+	var test = {
+		title: 'test \u001b[90m\u001b[2m›\u001b[22m\u001b[39m my test',
```

Comment:
Use `chalk` to colorize it. Makes it easier if we ever want to change the style.